### PR TITLE
Added page for describing teams and members

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

**Comment by SeanKilleen** [comment]
Does this belong in the docs? I'm unsure.

**Comment by OsirisTerje** [comment]
I need to link to something from the Release notes, so Developer Info section do have a Team Practices, and the Team tab - which has this information, sort of - is not visible to external users.  Therefore I thought it could be a suitable place.

**Comment by SeanKilleen** [comment]
@OsirisTerje yeah that's fair; it's at least fine as an interim place. 👍 I'll aim to get it reviewed & merged today.

**Comment by OsirisTerje** [comment]
@SeanKilleen Beside this, we should perhaps think about how to better display project related information like this.  Also, Team Discussions will be moved by May 8th.  Any thoughts or ideas you might have here would be awesome.

I'll fix the spelling errors here.

File: docs/articles/developer-info/The-Teams.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+|Name|Core Team|Framework|Engine|VS Extensions|Analyzer|Documentation|
+|--|--|--|--|--|--|--|
+|[Rob Prouse](https://github.com/rprouse) :canada:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Terje Sandstrom](https://github.com/osiristerje) :norway:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Joseph Musser](https://github.com/jnm2) :us:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Mikkel Nylander Bundgaard](https://github.com/mikkelbu) :denmark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:||:heavy_check_mark:||
+|[Sean Killeen](https://github.com/SeanKilleen) :us:|:heavy_check_mark:||||:heavy_check_mark:|:heavy_check_mark:|
+|[Steven Weerdenburg](https://github.com/stevenaw) :canada: |:heavy_check_mark:|:heavy_check_mark:|||||
+|[Ihor Sychevskyi](https://github.com/Arhell) :ukraine:||||||:heavy_check_mark:|
+|[Manfred Brands](https://github.com/manfred-brands) :australia: ||:heavy_check_mark:|||||
```
Comment (manfred-brands) [inline docs/articles/developer-info/The-Teams.md:58]: @OsirisTerje I should be listed in the analyzer team

Comment (OsirisTerje) [reply docs/articles/developer-info/The-Teams.md]: Done.   Also approved the Analyzer Team membership.

Comment (mikkelbu) [reply docs/articles/developer-info/The-Teams.md]: @OsirisTerje I know this has nothing to do with the PR, but given that you have more "power" than I have. Can you also make me the maintainer of the team (currently it is @SeanKilleen as he created the team).

Comment (OsirisTerje) [reply docs/articles/developer-info/The-Teams.md]: Sure!  and Done!

**Pull request merged**